OpenSim/Application.swift

```diff
@@ -24,12 +24,12 @@ final class Application {
     static let sizeDispatchQueue = DispatchQueue(label: "com.pop-tap.size", attributes: .concurrent, target: nil)
     
     var sandboxUrl: URL? {
-        guard let url = device.containerURLForApplication(self),
-            FileManager.default.fileExists(atPath: url.path)
+        guard let safeurl = device.containerURLForApplication(self),
+            FileManager.default.fileExists(atPath: safeurl.path)
             else {
-                return nil
+                return url
```

Comment:
My bad. I am using it primarily for deleting the apps, so my requirement is exposing the app url on the menu. Just ignore it.(On my mac it failed to achieve the sandbox url unless I booted the related device once, returning app url helps a bit.)